cornedbeef/src/main/java/com/swiftkey/cornedbeef/PunchHoleCoachMark.java

```diff
@@ -320,6 +326,16 @@ public PunchHoleCoachMarkBuilder setHorizontalTranslationDuration(long horizonta
             return this;
         }
 
+        /**
+         * Set the padding for the punch hole around the anchor view
```

Comment:
Can you explain how the padding appears? E.g. "the padding increases the punch hole's radius by N pixels" What are the units?